arlas-core/src/main/java/io/arlas/server/model/request/Page.java

```diff
@@ -19,7 +19,9 @@
 
 package io.arlas.server.model.request;
 
-public class Size {
+public class Page {
     public Integer size;
     public Integer from;
+    public String sort;
```

Comment:
If you only sort the result, it is strange to use a Page whereas you want all results, isn't it?

Reply:
There is a default `size` so you cannot have all sorted results without querying with an appropriate page size for the list you want to get. By the way, it has already been discussed in API meeting, no need to go back to discussion for the moment ;)